packages/test/test-service-load/src/stressTest.ts

```diff
@@ -113,42 +122,36 @@ export async function stressTest(
 		}, 20000);
 	}
 
-	try {
```

Comment:
@ChumpChief  why did you get rid of this try-finally?  If the callback to `runnerArgs.map` throws, the ending logs won't be logged to console.

Reply:
The throw should propagate up to the caller (`main()`) - there I have a try/finally where it should be caught and logged (well, at least the first failure will - the handling still isn't great in the case of multiple failures).  Are you seeing cases where errors don't get caught there?

Reply:
Followed up with Mark - this comment was referring to the logging of "end time" and "run time" at the end of the test.  I omitted these in the error case intentionally, as the test didn't successfully complete and so the "end time" is somewhat nonsensical.

Fair to make changes here going forward though - maybe with additionally clarifying the log text to explain if its referring to a successful run duration vs. how long it ran before erroring out.